### ISSUE TITLE
OverflowMenu (React) focus trap

### ISSUE THREAD

**Issue by Georgegriff** [carbon-design-system/carbon#4694]
http://react.carbondesignsystem.com/?path=/story/overflowmenu--basic
## Steps to reproduce 
Using the link above, click just above the "..." menu and press "tab". I'm finding that once the overflow menu is focused i'm unable to keep pressing tab to other components?

perhaps i'm missing some magic key press to get around tiis?

## Environment

> Operating system
Mac OS

> Browser

Chrome

**Comment by emyarod** [comment]
this sounds like a duplicate of https://github.com/carbon-design-system/carbon/issues/4659, addressed in https://github.com/carbon-design-system/carbon/pull/4688

**Comment by Georgegriff** [comment]
Yeah sounds like it, sorry I did do a search by not well enough clearly :) will close and follow the other issues
